contents/resources/dariah-de-publikator/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: DARIAH-DE Publikator Tutorial
+date: 2019-12-01
+authors:
+  - dariah-de
+contributors:
+editors:
+categories:
+  - dariah
+tags:
+  - research-infrastructures
+  - dh
+  - metadata
+  - open-science
+  - sustainability
+  - repositories-and-collections
+  - open-access
+abstract:
+"This tutorial explains the fundamentals of the DARIAH-DE Publikator, the entry point for importing collections and research data into the DARIAH-DE Repository, which allows you to prepare, manage, and finally import your collections into the DARIAH-DE Repository using your favourite internet browser.
+The tutorial contains guides for users as well as technical documentation. The user guides are  conveniently organized into three sections: Collections, Log in with a DARIAH-DE Account or with a Federation Account, and Publishing with the DARIAH-DE Publikator. Every section has examples to explain different aspects of the DARIAH-DE Publikator."
+
+type: training module
+remoteUrl: https://repository.de.dariah.eu/doc/services/submodules/publikator/docs/index.html
+remotePublicationDate: 2019-12-01
+---
+
+This tutorial explains the fundamentals of the DARIAH-DE Publikator, the entry
```

Comment:
lengthwise this is ok, but it should be split into two paragraphs, for easier reading

Reply:
also the first sentence should be fixed —  see previous comment